tsdb/internal/meta.pb.go

```diff
@@ -16,7 +16,7 @@ It has these top-level messages:
 */
 package internal
 
-import proto "github.com/golang/protobuf/proto"
```

Comment:
I meant to ask about this. So the existing import is the one we want, right? I want to be sure.

I will revert this part.